File: code/app/src/main/java/com/example/avtokampi_android_app/models/Avtokamp.kt
package com.example.avtokampi_android_app.models

import com.google.android.gms.maps.model.LatLng

data class Avtokamp(
    val id: Int,
    val naziv: String,
    val opis: String,
    val naslov: String,
    val telefon: String,
    val naziv_lokacije: String,
    val isActive: Boolean,
    val lokacija: LatLng,
    val createdAt: String,
    val updatedAt: String,
    val regija: Regija
)

File: avtokampi/lib/models/Avtokamp.dart
class Avtokamp {
    int id;
    String naziv;
    String opis;
    String naslov;
    String telefon;
    String nazivLokacije;
    String koordinataX;
    String koordinataY;
    bool isActive;
    String createdAt;
    String updatedAt;
    int regija;

    Avtokamp.nov();


    Avtokamp(this.id, this.naziv, this.opis, this.naslov, this.telefon,
        this.nazivLokacije, this.koordinataX, this.koordinataY, this.isActive,
        this.createdAt, this.updatedAt, this.regija);

    static Avtokamp fromJson(json) {
        Avtokamp a = new Avtokamp.nov();
        a.id = json['avtokampId'];
        a.naziv = json['naziv'];
        a.opis = json['opis'];
        a.naslov = json['naslov'];
        a.telefon = json['telefon'];
        a.nazivLokacije = json['nazivLokacije'];
        a.koordinataX = json['koordinataX'];
        a.koordinataY = json['isActive'];
        a.createdAt = json['createdAt'];
        a.updatedAt = json['updatedAt'];
        return a;
    }
}